core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -722,35 +723,47 @@ public static String getDbVersion() {
     }
     
     public static String getDarwinAuthCheckUrl() {
-        String darwinAuthUrl = properties.getProperty(DARWIN_AUTH_URL);
-        if (darwinAuthUrl == null || darwinAuthUrl.isEmpty()) {
-            return "";
+        String darwinAuthUrl;
+        try{
+            darwinAuthUrl = properties.getProperty(DARWIN_AUTH_URL).trim();            
         }
-        return darwinAuthUrl.trim();
+        catch (NullPointerException e){
```

Comment:
Another remark: it is confusing that `DARWIN_AUTH_URL` means something else here when compared to the same constant in `CheckDarwinAccessMain.java`. I would suggest removing the constant in `CheckDarwinAccessMain.java` and just using this getter method.